src/__tests__/realmRoles.test.ts

```diff
@@ -0,0 +1,77 @@
+import Account from '../account'
+// @ts-ignore no type defs exist for js-sdk
+import Tozny from '@toznysecure/sdk/node'
+import { v4 as uuidv4 } from 'uuid'
+import { cleanupRealms } from './utils'
+import { Role } from '../types'
+
+jest.setTimeout(10000)
```

Comment:
why is this needed, can we add a comment on the why to this file, move this wait into a test helper?

Reply:
added a comment for explanation. if more tests start requiring more time to run, i'll change the global default in the jest config.